### DISCUSSION TITLE
How to automate selecting file path & name in a "select file dialog"?

### DISCUSSION THREAD

**Discussion by GHRyunosuke** [GCuser99/SeleniumVBA#94] [Q&A] [answered]
Hello, guys, while automating Chrome to upload a file, as shown below, a "select file dialog" comes up after clicking the upload button. How shall I select the corresponding file path & name of the file I want to upload and then proceed? 

![image](https://github.com/GCuser99/SeleniumVBA/assets/25561705/a51db2c7-d656-40dc-9799-9386f6e1466b)

**Comment by GCuser99** [comment, marked as answer]
@GHRyunosuke, have you tried the UploadFile method of WebElement/WebDriver classes? There is an example module in the test set called "test_FileUpDownload" - see the "test_file_upload" sub...

**Comment by GHRyunosuke** [comment]
Thank you @GCuser99, I got the idea of the "test_FileUpDownload" example, let me try it out...

**Comment by GHRyunosuke** [comment]
@GCuser99 problem solved with the "test_FileUpDownload" example. I shall pay more attention to the examples folder. Thanks a lot !

**Comment by GCuser99** [comment]
@GHRyunosuke good to hear!
